cpp/include/cugraph/graph_view.hpp

```diff
@@ -286,6 +286,33 @@ class graph_base_t : public graph_envelope_t::base_graph_t /*<- visitor logic*/
 
 }  // namespace detail
 
+template <typename vertex_t, typename edge_t, bool multi_gpu, typename Enable = void>
```

Comment:
`graph_view_meta_t` seems repetitive of a functionality that `graph_meta_t` already offers. Is there a strong motivation for this apparent code duplication? Would it be possible to delegate this _grouping_ functionality to only one such `meta` pair?

Reply:
Yes, they are currently pretty much the same but this restructuring is mainly to pass additional parameters. When we support (key, value) pairs for row/column properties, pointers to unique rows/columns will be only passed to graph_view_meta_t.

So, these two will diverge (and possibly diverge even more over time), and that's the key motivation for having two separate classes.

Reply:
There will be an overlap but the two will have non-overlapping parts.